public/index.html

```diff
@@ -19,7 +19,7 @@ <h2>You are running a Swift Kitura microservice built for the IBM Cloud.</h2>
         <p>
             A microservice is an individual component of an application that follows the microservice architecture - an architectural style that structures an application as a collection of loosely coupled services, each of which implement a business capability. The microservice exposes a RESTful API matching a <a target="_blank" rel="noopener noreferrer" href="http://swagger.io">Swagger</a> definition.
         </p>
-        <b>This Node.js microservice has these endpoints:</b>
+        <b>This Kitura microservice has these endpoints:</b>
```

Comment:
This might break the skit experience test. Probably wont since it's not the main text.

Reply:
checked the script and it looks good